.github/workflows/pr-build.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
       - "release/v*"
+  workflow_dispatch:
```

Comment:
Do we want to run PR build in workflow_dispatch, as run async other than at the time of PR.